fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/navbar/NavbarIntegration.kt

```diff
@@ -18,17 +25,29 @@ import mozilla.components.support.base.feature.LifecycleAwareFeature
 class NavbarIntegration(
     val toolbar: ScrollableToolbar,
     val store: BrowserStore,
+    val appStore: AppStore,
+    val viewLifecycleOwner: LifecycleOwner,
+    val bottomToolbarContainerView: BottomToolbarContainerView,
```

Comment:
❓ Should we remove `toolbar: ScrollableToolbar` now that we're passing in `BottomToolbarContainerView` which includes the `ToolbarContainerView`.  If so I suggest a follow up to do this.